stable/rabbitmq-ha/templates/statefulset.yaml

```diff
@@ -146,6 +146,12 @@ spec:
             {{- end }}
           resources:
 {{ toYaml .Values.resources | indent 12 }}
+          {{- if .Values.securityContext.enabled }}
```

Comment:
Remove the if and just copy the entire securityContext via `{{- toYaml .Values.securityContext | nindent 10 }}`. That way additional fields can be customized If people want to disable the securityContext, they can always `--set securityContext=null` to disable.

Reply:
I've used indent instead of indent to stay consistent within the file.